src/sdk/models/diagnostics.ts

```diff
@@ -0,0 +1,5 @@
+import { IEntityIdentifier } from './base';
+
+export interface DiagnosticMetric extends IEntityIdentifier<string> {
+	metrics: any;
```

Comment:
I expected fields like name, value, etc here.
Also, is `IEntityIdentifier` inheritance needed? I don't recall seeing an `id` in the data.

Reply:
good point. I will look into the IEntityIdentifier and interface for the model.

Reply:
updated. removed inheritance from IEntityIdentifer and updated the model.